docs/INSTALL-COMPLETE-GUIDE.md

```diff
@@ -0,0 +1,145 @@
+1. Click `Write`
+1. Wait for the imaging process to be finished (it'll take a few minutes) and eject your SD card
+
+## Initial boot - Set up Wifi connection
+
+1. Connect a USB mouse, a keyboard and a screen through HDMI
+1. Insert the Micro SD card
+1. Start your Raspberry Pi by attaching a power supply
+1. Walk through the "Welcome to Raspberry Pi" wizard (the content of this wizard might have changed after this tutorial was created!)
+    1. Set your locale
+    1. Change your password
+    1. Proceed with with screen settings
+    1. Select your Wifi Network (You are being asked in the Phoniebox install routine whether you want to set up your Wifi again, we will skip this step then)
+    1. Software Update (optional, recommended, takes a while)
+    1. Reboot
+1. Let's enable a few more settings
+    1. Launch `Raspberry Pi Configuration` from the `Preferences` menu
+    1. A window opens with the `System` tab selected
+    1. Select `To CLI` for `Boot` option
+    1. Select `Login as user 'pi'` for `Auto login` option
+    1. Select `Wait for network` for `Network at Boot` option (optional, required for Spotify+ version)
+    1. Navigate to the `Interfaces` tab
+    1. Select `Enabled` next to `SSH`
+    1. Click `OK`
+1. Set up a static IP (optional, recommended)
+    1. Right click on the Wifi sympbol in the upper right corner of your application bar and choose `Wifi & Wired Network Settings`
+    1. Configure `interface` and `wlan0`
+    1. Check `Disabled IPv6` unless you want to provide a static IPv6 address
+    1. Fill out `IPv4` and `Router` (Gateway) options (keep `DNS Servers` and `DNS Search` empty)
+    1. Click `Apply` and `Close`
+1. Optional: If you like, you can **turn off Bluetooth** to reduce energy consumption (unless you want to use any Bluetooth devices with your Phoniebox)
+1. Shutdown your Raspberry Pi (`Application > Logout > Shutdown`)
+
+## Install Phoniebox software
+
+If you want to install the **Spotify+ version**, [read this first](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/develop/docs/SPOTIFY-INTEGRATION.md).
+
+1. While shut down, disconnect mouse, keyboard and HDMI
+1. Connect RFID Reader
+1. Conntect USB Sound Card, plug in the 3.5" speakers with the Ground Loop Isolator in between. If you have chosen the example speakers from above, you can power them either through the Raspberry Pi or through an external power source.
+1. Boot your Raspberry Pi
+1. Open a terminal in your second computer and login via SSH using the `pi` user and your static IP address. If you see a question about authentication and fingerprint, type `yes` and hit `enter`
+    ```
+    ssh pi@192.168.1.123
+    ```
+
+### Configure USB sound card
+
+1. Configure your **USB sound card**. Check if your sound card has been detected
+    ```
+    cat /proc/asound/modules
+
+    // returns
+
+    0 snd_bcm2835
+    1 snd_usb_audio
+    ```
+1. To update the sound card priority order, edit the following file
+    ```
```

Comment:
Is this really necessary?

Reply:
Good question... it shouldn't be needed, but this is exactly the problem that some Phonieboxers ran into when unplugging the HDMI